Assets/UI/worldinput_CQUI.lua

```diff
@@ -0,0 +1,559 @@
+-- ===========================================================================
+-- Base File
+-- ===========================================================================
+include("WorldInput");
+
+-- ===========================================================================
+-- Cached Base Functions
+-- ===========================================================================
+BASE_CQUI_UpdateDragMap            = UpdateDragMap;
+BASE_CQUI_RealizeMovementPath      = RealizeMovementPath;
+BASE_CQUI_OnUnitSelectionChanged   = OnUnitSelectionChanged;
+BASE_CQUI_OnInputActionTriggered   = OnInputActionTriggered;
+BASE_CQUI_DefaultKeyDownHandler    = DefaultKeyDownHandler;
+BASE_CQUI_DefaultKeyUpHandler      = DefaultKeyUpHandler;
+BASE_CQUI_OnDefaultKeyDown         = OnDefaultKeyDown;
+BASE_CQUI_OnDefaultKeyUp           = OnDefaultKeyUp;
+BASE_CQUI_OnPlacementKeyUp         = OnPlacementKeyUp;
+BASE_CQUI_ClearAllCachedInputState = ClearAllCachedInputState;
+BASE_CQUI_LateInitialize           = LateInitialize;
+BASE_CQUI_Initialize               = Initialize;
+
+-- ===========================================================================
+-- CQUI Members
+-- ===========================================================================
+local CQUI_HOTKEYMODE_ENHANCED    = 2;
+local CQUI_HOTKEYMODE_CLASSIC     = 1;
+local CQUI_HOTKEYMODE_STANDARD    = 0;
+
+local m_isInputBlocked  :boolean = false;
+local CQUI_cityview     :boolean = false;
+local CQUI_hotkeyMode   :number  = CQUI_HOTKEYMODE_ENHANCED;
+local CQUI_isShiftDown  :boolean = false;
+local CQUI_isAltDown    :boolean = false;
+local CQUI_isCtrlDown   :boolean = false;
```

Comment:
Doesn't seem to be used anywhere.

Reply:
yep, added it with intention of using it, then didn't use it after reviewing some more.
removed.